src/unity/python/turicreate/toolkits/image_classifier/_annotate.py

```diff
@@ -0,0 +1,216 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+"""
+Class definition and utilities for the annotation utility of the image classification toolkit
+"""
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+
+from ...visualization import _get_client_app_path
+
+import turicreate as __tc
+
+def _warning_annotations():
+    print(
+        """
+        *** WARNING: If you don't assign the output of the annotate function
+                     to a variable all annotations are going to be lost.
+                     To recover the last annotated SFrame call:
+
+        `tc.image_classifier._recover_annotation()`
+
+        """
+    )
+
+def annotate(data, image_column = 'image', annotation_column = 'annotations'):
```

Comment:
In the image classifier the default value for the image column is `None` and we infer the image column (if there is only one column):
https://github.com/apple/turicreate/blob/5aacc9ceabae8bae5536dcafc6a652c8cfbc7b05/src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py#L253
That would be nice to do here too.